### PR TITLE
2.second commit after review

### PR DESCRIPTION
**Pull request by Muhammed-fazil-k**
2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,83 +1,99 @@
...
-```
-<repositories>
-		<repository>
-		    <id>jitpack.io</id>
-		    <url>https://jitpack.io</url>
-		</repository>
-	</repositories>
 ```
-Add a dependency
+3. For using MaterialLetterIcon from a remote repository in separate application, add the below dependencies in entry/build.gradle file.
```
Comment (samuel-abhishek) [inline README.md]: This needs to be removed. Must be added after Maven release

File: .idea/gradle.xml [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="distributionType" value="DEFAULT_WRAPPED" />
+        <option name="testRunner" value="PLATFORM" />
+        <option name="distributionType" value="LOCAL" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="gradleHome" value="D:\APP\deveco\devecostudio-windows-tool-2.1.0.303copy\DevEco Studio 2.1.0.303\tools\gradle" />
         <option name="modules">
           <set>
```
Comment (samuel-abhishek) [inline .idea/gradle.xml]: .idea folder needs to be deleted.

File: README.md (hunk 1 above)
Comment (Muhammed-fazil-k) [reply README.md]: removed that part

**Review by samuel-abhishek — APPROVED** [review]
LGTM

**Pull request merged**